stream-chat-android-client/src/test/java/io/getstream/chat/android/client/api/ClientConnectionTests.kt

```diff
@@ -84,19 +82,11 @@ internal class ClientConnectionTests {
         val tokenUtils: TokenUtils = mock()
         whenever(tokenUtils.getUserId(token)) doReturn userId
         socket = mock()
-        retrofitApi = mock()
-        retrofitAnonymousApi = mock()
         fileUploader = mock()
         logger = mock()
         notificationsManager = mock()
         initCallback = mock()
-        api = GsonChatApi(
-            retrofitApi,
-            retrofitAnonymousApi,
-            UuidGeneratorImpl(),
-            fileUploader,
-            testCoroutines.scope
-        )
+        api = mock()
```

Comment:
Mocking the entire `ChatApi` now instead of its contained Retrofit API interfaces, this way we don't have to create dummy DTOs and deal with mapping in our tests which would be very, very tedious